value/src/main/java/com/google/auto/value/processor/autovalue.vm

```diff
@@ -35,7 +35,7 @@ final class $subclass$formalTypes extends $origClass$actualTypes {
   $subclass(
 #foreach ($p in $props)
 
-      $p.type $p #if ($foreach.hasNext) , #end
+     #if ($p.nullable) ${p.nullableAnnotation} #end $p.type $p #if ($foreach.hasNext) , #end
```

Comment:
It seems ok. 
I put a break point in AutoValueProcessor.processType to verify manually while executing the unit tests. Let me know if this is not the right way to verify this or if special test cases should be added.